marquez_client/file_backend.py

```diff
@@ -54,7 +55,13 @@ def post(self, path, headers, json=None):
 
         log.info(post_details)
 
-        self._file.write(f'{post_details}{os.linesep}')
+        self.sync_file(post_details)
+
+    def sync_file(self, json_data):
```

Comment:
I would make the method internal: `_sync_file`

Reply:
done.